code/nebula/neb.h

```diff
@@ -73,6 +73,10 @@ typedef struct poof_info {
 	::util::UniformFloatRange rotation;
 	float view_dist;
 	::util::UniformFloatRange alpha;
+	int fade_start;
+	int fade_duration;
+	bool fade_in;
+	float fade_multiplier;
```

Comment:
The only thing I would say, is to separate these values from the others a bit and comment that they are dynamic values which can vary during a mission, unlike the others which are always completely static,

Reply:
Good idea. Done.